mturk/views.py

```diff
@@ -0,0 +1,11 @@
+from django.shortcuts import render
+
+
+def mturk_index(request, *args, **kwargs):
+    from mturk.interface import MTurkProvider
+    from crowdsourcing.models import Project, Task
+    provider = MTurkProvider('https://localhost:8000')
+    project = Project.objects.get(id=1)
```

Comment:
Should be using SITE_HOST instead of localhost
create_external_question uses taskId which is not used here.